lib/foreach.coffee

```diff
@@ -9,8 +9,10 @@ regEx = require './regex'
 
 module.exports = (options)-> new Promise (finish)->
 	finalLogs = 'log':{}, 'error':{}
+	globOptions = {}
+	if options.ignore then globOptions.ignore = options.ignore
```

Comment:
Looks good, but I think this can be reduced to a single `globOptions = {ignore:options.ignore}` which will resolve to the same logic

Reply:
Thanks for merging, want me to fix in new PR?